wercker-step.yml

```diff
@@ -1,3 +1,3 @@
 name: rails-database-yml
-version: 1.0.2
```

Comment:
i see :)
So, how about do these?

- Separate branch between PR to original one and Temporary we use one which is based on PR to original one.
- Add suffix like `-alpha` when you want to test. 
   - http://semver.org/